starter-code/src/viking.js

```diff
@@ -1,11 +1,87 @@
 // Soldier
-function Soldier() {}
+function Soldier(healthArg, strengthArg) {
+  (this.health = healthArg), (this.strength = strengthArg);
+}
```

Comment:
function Soldier(healthArg, strengthArg) {
  this.health = healthArg;
  this.strength = strengthArg;
}